plugin/trino-hive/src/main/java/io/trino/plugin/hive/BackgroundHiveSplitLoader.java

```diff
@@ -472,6 +473,8 @@ private void invokeNoMoreSplitsIfNecessary()
 
             JobConf jobConf = toJobConf(configuration);
             FileInputFormat.setInputPaths(jobConf, path);
+            // SerDes parameters and Table parameters passing into input format
```

Comment:
change comment: `Pass SerDes and Table parameters into input format configuration`

Reply:
done